precommit.sh

```diff
@@ -8,3 +8,5 @@ docker exec tcf_django node_modules/.bin/eslint --cache --fix -c .config/.eslint
 docker exec tcf_django coverage run manage.py test --keepdb
 # Get code coverage.
 docker exec tcf_django coverage report -m
+
+python3 seleniumtests/selenium_test.py
```

Comment:
I also know very little about Selenium, but generally I believe we want everything to be run in the Docker container (i.e. put `docker exec tcf_django` before all the commands). If running the tests outside Docker is is a deliberate choice on your part, please ignore!

Reply:
Yes, I wrote about this in my concerns. Thank you for giving me the actual command to run this lol